src/commands/Google/weather.ts

```diff
@@ -18,18 +19,23 @@ const COLORS = {
 	windy: '#33B679'
 };
 
-export default class extends SkyraCommand {
-	public constructor(store: CommandStore, file: string[], directory: string) {
-		super(store, file, directory, {
-			bucket: 2,
-			cooldown: 120,
-			description: (language) => language.get('commandWeatherDescription'),
-			extendedHelp: (language) => language.get('commandWeatherExtended'),
-			requiredPermissions: ['ATTACH_FILES'],
-			usage: '<city:string>'
-		});
-	}
+const celsiusToFahrenheit = (celsius: number) => celsius * 1.8 + 32;
```

Comment:
```suggestion
const celsiusToFahrenheit = (celsius: number) => (celsius * 9 / 5) + 32;
```

9 / 5 is 1.8 yes, but the official formula has `9 / 5` which is the official conversion (and because 5 / 9 for the other way around is not roundable (0.55555555555555555555556 basically))

![image](https://user-images.githubusercontent.com/4019718/94301700-545a2980-ff6b-11ea-81c9-37de1f5dcc86.png)

Reply:
While deriving the actual formula, you multiply it by `180 / 100` which is simplified to `9/5`. So 1.8 should be fine.
[Source](https://www.mathsisfun.com/temperature-conversion.html)

Reply:
We prioritise precision over speed, which is why our saturation calculation is the complex variant.